forge-gui/res/cardsfolder/upcoming/riku_of_many_paths.txt

```diff
@@ -0,0 +1,17 @@
+Name:Riku of Many Paths
+ManaCost:G U R
+Types:Legendary Creature Human Wizard
+PT:3/3
+T:Mode$ SpellCast | ValidActivatingPlayer$ You | ValidSA$ Spell.Modal | Execute$ TrigCharm | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a modal spell, ABILITY
+SVar:TrigCharm:DB$ Charm | AdditionalDescription$ where X is the number of times you chose a mode for that spell | Choices$ DBExile,DBPutCounter,DBToken | CharmNum$ X | MinCharmNum$ 0
+SVar:X:TriggeredSpellAbility$NumTimesChoseMode
+SVar:DBExile:DB$ Dig | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top card of your library. Until the end of your next turn, you may play it.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | ExileOnMoved$ Exile | RememberObjects$ Remembered | Duration$ UntilTheEndOfYourNextTurn | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ Until the end of your next turn, you may play it.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBPutCounter:DB$ PutCounter | CounterType$ P1P1 | SubAbility$ DBPump | SpellDescription$ Put a +1/+1 counter on CARDNAME.
+SVar:DBPump:DB$ Pump | KW$ Trample | StackDescription$ SpellDescription | SpellDescription$ It gains trample until end of turn.
+SVar:DBToken:DB$ Token | TokenScript$ u_1_1_bird_flying | SpellDescription$ Create a 1/1 blue Bird creature token with flying.
```

Comment:
@Northmoc
looks like this splitting of SpellDescription isn't supported:
![image](https://github.com/Card-Forge/forge/assets/8506892/8d8ea6ae-d022-4041-a5e9-34e20de4fdc7)